### ISSUE TITLE
Data Privacy Info Sheet: Add reference to Terms of Service

### ISSUE THREAD

**Issue by DerekCaelin** [techmatters/terraso-mobile-client#2663]
Add reference to Terrsao terms of service to the Data Privacy Info Sheet

#### Screen to change
<img src="https://github.com/user-attachments/assets/b0733c78-c8c3-4db4-9fed-7135d2001dcf" width="200px">

## Acceptance Criteria
- [ ] String change: "Read more about our data privacy policy." -> "Read more about our [terms of service](https://terraso.org/terms-of-service/)  and [privacy policy](https://techmatters.org/privacy-policy/)."

**Comment by johannesparty** [comment]
The existing layout is a list of paragraphs and external and internal link (external one has the little arrow and all caps).  Following same system, we would have something like this:

```
Read more about our 
[terms of service]
and 
[privacy policy]
```

If you want this ... internal or external link style?

If you want inline text that is doable too ... blue (common) or green (shown above)?

**Comment by DerekCaelin** [comment]
External link, please!

**Comment by DerekCaelin** [comment]
Ah. @CourtneyLee333 please weigh in here.

**Comment by CourtneyLee333** [comment]
Yeah, I think those designs were made before we implemented link components. @DerekCaelin any time we decide to take on a UI implementation task I need to review the design before passing to the engineer. 

@johannesparty The external link buttons are great. Please just get rid of "Read more about".

**Comment by CourtneyLee333** [comment]
Ah, we were expecting the links to open in webview, so they'd appear as internal links, but we haven't implemented webview.

**Comment by DerekCaelin** [comment]
This looks fine to me. @CourtneyLee333?

**Comment by DerekCaelin** [comment]
@johannesparty would you please update the string? Since it's such a minor change, no need to resubmit for Courtney and I to re-review.

**Comment by johannesparty** [comment]
Will do!

Johannes Schmidt
***@***.***
408-621-0091
